skycoin/dmsg
# Pty improvements

Fixes #57

 Changes:	
- moved the whitelist to config file code from dmsgpty-cli to dmsgpty-host
- added `NewConfigWhitelist()` to dmsgpty/whitelist.go for the host to read wl from the config
- added `configWhitelist` struct and the methods `All()`,  `Add()`, `Remove()` and `open()`
- removed viper dependency by merging the code from the PR #60 
- added instructions for dmsgpty in docs
- updated `github.com/creack/pty v1.1.9` to `github.com/creack/pty v1.1.10` because it was giving the following eror `Setctty set but Ctty not valid in child`

How to test this PR:
1. Run `make build`
2. Run `./bin/dmsgpty-host confgen`
3. Run `./bin/dmsgpty-host`
4. Run `./bin/dmsgpty-cli whitelist` or  `./bin/dmsgpty-cli whitelist-add public-key-goes-here` or `./bin/dmsgpty-cli whitelist-remove public-key-goes-here` on a different terminal
5. Run `./bin/dmsgpty-ui` and check `http://127.0.0.1:8080/`
